src/assets/styles/base/_base.scss

```diff
@@ -28,25 +28,53 @@ a {
   text-decoration: underline;
 }
 
-a:hover,
-a:focus {
-  background: var(--fl-linkColor, var(--accent, var(--yellow-500)));
-  color: var(--fl-bgColor, currentColor);
-  outline: none;
+main,
+footer {
+  a {
+    border-radius: 5px;
+  }
+
+  a:hover,
+  a:focus,
+  a:active {
+    outline-style: solid;
+    outline-width: 4px;
+  }
+
+  a:hover {
+    background: var(--fl-linkColor, var(--accent, var(--yellow-500)));
+    color: var(--fl-bgColor, currentColor);
+    outline-color: var(--fl-linkColor, var(--accent, var(--yellow-500)));
+  }
+
+  a:focus {
+    outline-color: var(--fl-linkColor, var(--indigo-800));
+    outline-offset: 2px;
+    outline-width: 2px;
+  }
+
+  a:active {
+    background: var(--fl-linkColor, var(--indigo-800));
+    color: var(--fl-bgColor, var(--white));
+    outline-color: var(--fl-linkColor, var(--indigo-800));
+  }
+}
+
+a[hreflang],
+a[download],
+a[rel='external'] {
+  svg {
+    display: inline-block;
+    margin-left: 0.25em;
+    margin-right: 0.25em;
```

Comment:
With updated focus style, it looks little off to have margin on the svg:

<img width="264" alt="Screen Shot 2023-06-21 at 1 24 03 PM" src="https://github.com/inclusive-design/idrc/assets/22668696/c44425d4-e043-40af-a144-9f165a9fffb2">

As shown in the image, the link has no space to the left but has some space to the right because of the margin. I am thinking, could we remove the margin on SVG, and make the wrapper to handle the gap? Here's an example:

```
a[rel='external'] {
  display: inline-flex;
  flex-direction: row;
  gap: 0.1rem;
}
```